tests/jballerina-unit-test/src/test/resources/test-src/bala/test_bala/object/test_bir_negative.bal

```diff
@@ -0,0 +1,20 @@
+import bir/objs;
+
+function f1(objs:Bar bar) {
+    assertEquality(bar is objs:Foo, false);
+}
+
+function f2(objs:Foo foo) {
+    assertEquality(foo is objs:Bar, false);
+}
+
+function f3(objs:Xyz xyz) {
+    assertEquality(xyz is objs:Qux, false);
+}
+
+function f4(objs:Qux qux) {
+    assertEquality(qux is objs:Xyz, false);
+}
+
+function assertEquality(any|error actual, any|error expected) {
```

Comment:
Is this and calls necessary for this test?

Reply:
Yes. We need to check after the BIR changes whether there any issues w.r.t objectTypes.

Reply:
I think Maryam is questioning the fact that since this specific sample fails at CT and the only validation happening when calling `assertEquality` is checking whether those object types are assignable to `any|error`; which is not actually needed.
I think you can update the samples in the below manner
```ballerina
function f1(objs:Bar bar) {
    _ = bar is objs:Foo;
}
```

Reply:
Addressed [#90c5ee9](https://github.com/ballerina-platform/ballerina-lang/pull/40901/commits/90c5ee9af6208cf463782cea723e06a708c823fd).